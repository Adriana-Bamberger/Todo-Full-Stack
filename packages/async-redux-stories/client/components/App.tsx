import ErrorMessage from './ErrorMessage.tsx'
import LoadSubreddit from './LoadSubreddit.tsx'
import SubredditList from './SubredditList.tsx'
import WaitIndicator from './WaitIndicator.tsx'

function App() {
  return (
    <div className="app">
      <ErrorMessage />
      <LoadSubreddit>
        <WaitIndicator />
      </LoadSubreddit>
      <SubredditList />
    </div>
  )
}

export default App
