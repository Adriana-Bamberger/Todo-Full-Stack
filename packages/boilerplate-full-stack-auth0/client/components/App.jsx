import Nav from './Nav'
import PingRoutes from './PingRoutes'
import Registration from './Registration'
import Users from './Users'
import { Routes, Route } from 'react-router-dom'
import { useAuthCache } from '../auth0-utils'

function App() {
  useAuthCache()

  return (
    <div className="app">
      <Routes>
        <Route path="/" element={<Nav />} />
        <Route path="/" element={<Users />} />
        <Route path="/" element={<PingRoutes />} />
        <Route path="/profile" element={<Registration />} />
      </Routes>
    </div>
  )
}

export default App
