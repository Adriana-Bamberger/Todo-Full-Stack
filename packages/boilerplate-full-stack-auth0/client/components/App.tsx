import Nav from './Nav.tsx'
import PingRoutes from './PingRoutes.tsx'
import Registration from './Registration.tsx'
import Users from './Users.tsx'
import { Routes, Route } from 'react-router-dom'
import { useAuthCache } from '../auth0-utils.ts'

function App() {
  useAuthCache()

  return (
    <div className="app">
      <Routes>
        <Route path="/" element={<Nav />} />
        <Route path="/" element={<Users />} />
        <Route path="/" element={<PingRoutes />} />
        <Route path="/profile" element={<Registration />} />
      </Routes>
    </div>
  )
}

export default App
