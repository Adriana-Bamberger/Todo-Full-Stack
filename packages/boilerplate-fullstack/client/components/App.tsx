import { useEffect } from 'react'
import { useAppDispatch, useAppSelector } from '../hooks.ts'

import { fetchFruits } from '../actions/index.ts'

function App() {
  const fruits = useAppSelector((state) => state.fruits)
  const dispatch = useAppDispatch()
  useEffect(() => {
    dispatch(fetchFruits())
  }, [dispatch])

  return (
    <>
      <div className="app">
        <h1>Fullstack Boilerplate - with Fruits!</h1>
        <ul>
          {fruits.map((fruit) => (
            <li key={fruit}>{fruit}</li>
          ))}
        </ul>
      </div>
    </>
  )
}

export default App
