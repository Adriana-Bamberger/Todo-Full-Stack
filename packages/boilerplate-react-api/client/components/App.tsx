import { useState, useEffect } from 'react'
import { getGreeting } from '../apiClient.ts'
import { useQuery } from '@tanstack/react-query'

const App = () => {
  const [count, setCount] = useState(0)

  const {
    data: greeting,
    isError,
    isLoading,
  } = useQuery({ queryKey: ['greeting', count], queryFn: getGreeting })

  if (isLoading) return <p>Loading...</p>

  return (
    <>
      {count}
      <h1>{greeting}</h1>
      {isError && (
        <p style={{ color: 'red' }}>
          There was an error retrieving the greeting.
        </p>
      )}
      <button onClick={() => setCount(count + 1)}>Click</button>
    </>
  )
}

export default App
