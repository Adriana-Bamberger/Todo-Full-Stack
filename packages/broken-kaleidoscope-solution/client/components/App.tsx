import Pixel from './Pixel'

function App() {
  return (
    <>
      {Array.from({ length: 100 }).map((_, i) => (
        <Pixel key={i} />
      ))}
    </>
  )
}

export default App
