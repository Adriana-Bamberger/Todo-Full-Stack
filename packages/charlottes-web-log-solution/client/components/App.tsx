import Header from './Header'
import headerProps from '../data/header'

import Footer from './Footer'
import footerProps from '../data/footer'

import Posts from './Posts'
import OtherBlogs from './OtherBlogs'
import RecentEntries from './RecentEntries'

function App() {
  return (
    <div className="app">
      <Header {...headerProps} />
      <div className="body-container">
        <OtherBlogs />
        <Posts />
        <RecentEntries />
      </div>
      <Footer {...footerProps} />
    </div>
  )
}

export default App
