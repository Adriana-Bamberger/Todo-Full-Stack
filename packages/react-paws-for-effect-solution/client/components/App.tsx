import Dog from './Dog'
import Subtitle from './Subtitle'

function App() {
  return (
    <div className="container">
      <img
        className="spinner"
        alt="a spinning paw-print"
        src="/images/paw.png"
      />
      <Subtitle text="Doggos" />
      <Dog name="Fido" breed="Bulldog" superpower="plays fetch" />
      <Dog name="Bruce" breed="Dachshund" superpower="barks" />
    </div>
  )
}

export default App
