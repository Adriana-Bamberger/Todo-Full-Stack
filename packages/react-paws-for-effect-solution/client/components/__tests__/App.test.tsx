// @vitest-environment jsdom
import { describe, it, expect } from 'vitest'
import { render } from '@testing-library/react/pure'

import App from '../App.tsx'

describe('<App />', () => {
  it('renders how we expect', () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })
})
