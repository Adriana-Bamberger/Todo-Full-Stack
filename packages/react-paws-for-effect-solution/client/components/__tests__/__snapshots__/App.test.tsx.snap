// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders how we expect 1`] = `
<div>
  <div
    class="container"
  >
    <img
      alt="a spinning paw-print"
      class="spinner"
      src="/images/paw.png"
    />
    <h2>
      Doggos
    </h2>
    <div
      class="dog-wrapper"
    >
      <div
        class="dog"
      >
        <div
          class="dog-name-plate"
        >
          <span
            class="dog-name"
          >
            Fido
          </span>
          <span
            class="dog-breed"
          >
            Bulldog
          </span>
        </div>
        <span
          class="dog-superpower"
        >
          plays fetch
        </span>
        <img
          alt="a Bulldog"
          src="/images/bulldog.png"
        />
      </div>
    </div>
    <div
      class="dog-wrapper"
    >
      <div
        class="dog"
      >
        <div
          class="dog-name-plate"
        >
          <span
            class="dog-name"
          >
            Bruce
          </span>
          <span
            class="dog-breed"
          >
            Dachshund
          </span>
        </div>
        <span
          class="dog-superpower"
        >
          barks
        </span>
        <img
          alt="a Dachshund"
          src="/images/dachshund.png"
        />
      </div>
    </div>
  </div>
</div>
`;
